release-1.8/release_notes_draft.md

```diff
@@ -128,6 +132,15 @@ kind.
  in place of [Opaque Integer Resources](https://v1-6.docs.kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#opaque-integer-resources-alpha-feature).
  Extended resources cannot be overcommitted, so request and limit must be equal
  if both are present in a container spec.
+ - The default Bootstrap Token created with `kubeadm init` v1.8 expires
+ and is deleted after 24 hours by default to limit the exposure of the
+ valuable credential. It is straightforward to create a new Bootstrap
+ Token with `kubeadm token create` afterwards, or alternatively, you can
+ specify `--token-ttl 0` to `kubeadm init` to make the default token valid
+ forever (it can be deleted afterwards with `kubeadm token delete`, though)
```

Comment:
You can create a new Bootstrap Token with `kubeadm token create` or 
 make the default token permanently valid by setting `--token-ttl 0` to `kubeadm init`. The default token can later be deleted with `kubeadm token delete`.